src/common/components/with-store-subscription.tsx

```diff
@@ -15,6 +15,7 @@ export type WithStoreSubscriptionDeps<T> = {
     storeActionMessageCreator: IStoreActionMessageCreator;
 };
 
+// tslint:disable-next-line: typedef
```

Comment:
I think `React.ComponentClass<P, S>` is the return type you're looking for here

Reply:
Thank you. Worked like a charm.